packages/headless/src/app/engine.ts

```diff
@@ -127,12 +127,14 @@ export function buildEngine<
     platformUrl,
     analytics,
   } = options.configuration;
+  const name = options.configuration.name || 'coveo-headless';
 
   engine.dispatch(
     updateBasicConfiguration({
       accessToken,
       organizationId,
       platformUrl,
+      name,
```

Comment:
Why is the name part of the basic configuration? I think using the value when creating the initial store should be enough?